src/Client/OfficeInterop/OfficeInterop.fs

```diff
@@ -1096,12 +1096,12 @@ let addBuildingBlock (excelTable:Table) (arcTable:ArcTable) (newBB:CompositeColu
             let calIndex =
                 if targetIndex >= 0 then targetIndex + (float) i
                 else -1
+            log("targetIndex", targetIndex)
```

Comment:
One of:

- Remove logs
- Make logs more declarative. Something like `log ("[AddBioldnigBlock] - Excel - <placeholder>")`

If you decide to make them more declarative write a excellog function which you can reuse for this purpose

Reply:
fixed